go/vm/lfvm/ct.go

```diff
@@ -77,8 +77,7 @@ func (a ctAdapter) StepN(state *st.State, numSteps int) (*st.State, error) {
 		memory:      memory,
 		status:      RUNNING,
 		code:        converted,
-		isBerlin:    params.Revision >= vm.R09_Berlin,
-		isLondon:    params.Revision >= vm.R10_London,
+		revision:    params.Revision,
 		return_data: state.LastCallReturnData.ToBytes(),
 	}
```

Comment:
I see that the new enums are more robust, but the comparison <= or ==  is used quite often later on.
Instead of comparing the enum every time, potentially writting it wrong. Would it make sense to make a function (either member or freestanding) that returns bool whenever the context is Berlin or London capable? This way the comparison is written once. But the option to write a switch somewhere down the line is still there.